app/dts/bindings/animations/zmk,animation-ripple.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright (c) 2020, The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+description: |
+  Animated 'ripples' trigered by key presses.
+
+compatible: "zmk,animation-ripple"
+
+include: animation_base.yaml
+
+properties:
+  duration:
+    type: int
+    required: false
+    default: 1000
+    description: |
+      Approximate ripple travel time in milliseconds.
+
+  color:
+    type: int
+    required: true
+    description: |
+      Ripple color.
```

Comment:
I think we should add more detail here, specifically that the ripple color is specified in HSL